DataConnectors/Symantec Endpoint Protection/Connector_Syslog_SymantecEndpointProtection.json

```diff
@@ -0,0 +1,123 @@
+{ 
+    "id": "SymantecEndpointProtection",
```

Comment:
Per discussion with @preetikr, we decided not to add Broadcom to the name of this connector because of the customer familiarity with the name Symantec Endpoint Protection and in keeping of a shorter name